Mathlib/CategoryTheory/Topos/Classifier.lean

```diff
@@ -85,10 +85,10 @@ structure Classifier where
   χ₀ (U : C) : U ⟶ Ω₀
   /-- For any monomorphism `U ⟶ X`, there is an associated characteristic map `X ⟶ Ω`. -/
   χ {U X : C} (m : U ⟶ X) [Mono m] : X ⟶ Ω
-  /-- `χ m` and `χ₀ m` form the appropriate pullback square. -/
-  isPullback' {U X : C} (m : U ⟶ X) [Mono m] : IsPullback m (χ₀ U) (χ m) truth
+  /-- `χ₀ U` and `χ m` form the appropriate pullback square. -/
+  isPullback {U X : C} (m : U ⟶ X) [Mono m] : IsPullback m (χ₀ U) (χ m) truth
   /-- `χ m` is the only map `X ⟶ Ω` which forms the appropriate pullback square for any `χ₀'`. -/
-  uniq' {U X : C} (m : U ⟶ X) [Mono m] (χ₀' : U ⟶ Ω₀) (χ' : X ⟶ Ω)
+  uniq {U X : C} (m : U ⟶ X) [Mono m] {χ₀' : U ⟶ Ω₀} (χ' : X ⟶ Ω)
```

Comment:
```suggestion
  uniq {U X : C} (m : U ⟶ X) [Mono m] {χ₀' : U ⟶ Ω₀} {χ' : X ⟶ Ω}
```
I think we can even make `χ'` implicit here. since everything appears in `hχ'̀